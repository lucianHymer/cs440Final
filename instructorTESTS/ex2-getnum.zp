//  Purpose: Test for getnum()
//  input: your SD [last 5 digits] -- team member #1
//  input: your SD -- team member #2
//  Expected output: two numbers

int main () 
{
  int m1,m2;
  puts("Enter the last five digits of SID -- first member:\n");
  m1 = getnum();
  puts("Enter the last five digits of SID -- second member:\n");
  m2 = getnum();
  putn(m1);
  putn(m2);
  return 0;
}
