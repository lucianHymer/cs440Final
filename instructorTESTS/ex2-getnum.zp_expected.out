  201512
  201712
